source/ReproExamples/Issue514.tsx

```diff
@@ -52,10 +75,34 @@ export const Issue514 = () => {
   );
 };
 
-function SliderTrackMark({ isTrue }: TrackMarksProps) {
+export const Issue514 = () => {
+  return (
+    <View
+      style={{
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      <CustomizedSlider
```

Comment:
Here the Slider component has the name different, however, in released version the usage will still be based on `<Slider />`, just some more props will be available allowing to customize it.